packages/flutter_localizations/lib/src/l10n/cupertino_pt_PT.arb

```diff
@@ -0,0 +1,22 @@
+{
+  "todayLabel": "Hoje",
+  "datePickerHourSemanticsLabelOne": "$hour",
+  "datePickerHourSemanticsLabelOther": "$hour",
+  "datePickerMinuteSemanticsLabelOne": "1 minuto",
+  "datePickerMinuteSemanticsLabelOther": "$minute minutos",
+  "datePickerDateOrder": "dmy",
+  "datePickerDateTimeOrder": "date_time_dayPeriod",
+  "anteMeridiemAbbreviation": "AM",
+  "postMeridiemAbbreviation": "PM",
+  "alertDialogLabel": "Alerta",
+  "timerPickerHourLabelOne": "hora",
+  "timerPickerHourLabelOther": "horas",
+  "timerPickerMinuteLabelOne": "min",
+  "timerPickerMinuteLabelOther": "min",
+  "timerPickerSecondLabelOne": "seg",
```

Comment:
```suggestion
  "timerPickerSecondLabelOne": "s",
```

Reply:
Seems like you're suggesting opposite things between here and the same label in the previous file.

Reply:
Yes, for some reason the iOS abbreviates seconds in different ways in pt and pt_PT.

I changed my iPhone to pt_PT, accessed the Timer (in the clock App), and used the strings I found there.